### ISSUE TITLE
Reverse reads not simulated?

### ISSUE THREAD

**Issue by ozagordi** [sej917/BEAR#1]
Hi,
I uses the scripts to simulate reads from a pair of sequences as follows

```
python2.7 /usr/local/BEAR/scripts/generate_reads.py -r genomes.fasta -a abundances.txt -o meta.fasta -t 100000 -l 150
/usr/local/BEAR/scripts/trim_reads.pl -i real_reads.fastq -f meta.fasta -o simu.fastq -r 0 -q 0 -m 0
```

Turns out that all reads are generated in forward orientation. Unless I'm doing something wrong, which may well be, I would consider adding a switch to simulate from both orientations. Or at least making it clear in the documentation (I couldn't find it).

Thanks.

**Comment by sej917** [comment]
Sorry for taking so long to notice this! I had this functionality in a local copy of the script, and I just added it to the repo. You can simulate reads in both orientations with the -d flag.

**Comment by ozagordi** [comment]
Thanks,
but I'm afraid your patch only allows the generation of reverse, not reverse complements. This causes half the reads not being mapped.

**Comment by keithmewis** [comment]
Just to further on this; I just started using BEAR (downloaded June 3, 2015) and it seems the direction flag mentioned here has an error somewhere:

kmewis$ python generate_reads.py -r referencegenomes.fsa -a abundance_file -o output_fasta -t 500000 -l 580 -i 300
Traceback (most recent call last):
  File "generate_reads.py", line 18, in <module>
    parser.add_argument('-d', metavar='<direction_switch>', action='store_true', dest="direction", help="Use this switch to generate reads in both forward and reverse orientations" )
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/argparse.py", line 1294, in add_argument
    action = action_class(**kwargs)
TypeError: **init**() got an unexpected keyword argument 'metavar'

I commented out the -d in args and changed any reference to it to satisfy True (ie: permanently turning on the -d flag I guess) and it ran through but generated a 0 kb file for the reads.2.fasta. I'm a beginner level programmer so I'm not quite sure how to hunt down a solution for this.

Any thoughts on how to fix this?
